src/org/zaproxy/zap/extension/zest/ZestZapRunner.java

```diff
@@ -292,13 +277,13 @@ public String handleAssignment(ZestScript script, ZestAssignment assign, ZestRes
 
 	@Override
 	public void handleResponse(ZestRequest request, ZestResponse response) throws ZestAssertFailException {
-    	log.debug("handleResponse " + request.getElementType());
-	    try {
+		log.debug("handleResponse " + request.getElementType());
+		try {
 			HttpMessage msg = ZestZapUtils.toHttpMessage(request, response);
-			
-			ZestResultWrapper zrw = new ZestResultWrapper(Model.getSingleton().getSession(), 

```

Comment:
Please submit white space and formatting changes separately:
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting